src/game/Player.java

```diff
@@ -48,12 +48,18 @@ public int getXp() {
         return this.xp;
     }
 
-    public void setXp(int xp) {
-        this.xp = xp;
+    public void changeXp(int delta) {
+        this.xp += delta;
     }
 
-    public double getGoldCoins() {
-        return goldCoins;
+    public int getGoldCoins() {
+        return this.goldCoins;
+    }
+
+    private void changeGoldCoins(double delta) {
```

Comment:
This should be a public method
```suggestion
    public void changeGoldCoins(double delta) {
```